defaults/main.yml

```diff
@@ -1,10 +1,13 @@
 ---
 # Singularity target version
-singularity_version: "3.0.3"
+singularity_version: "3.6.0"
 
 singularity_conf_path: "/usr/local/etc/singularity/singularity.conf"
 
 bind_paths: []
 # example
 # - /shared/home/
 # - /shared/project/
+
+#go variables
+only_one_go_version: true
```

Comment:
Can you switch it to false to not change the previous behaviour?

Reply:
sure

Reply:
sure